src/public/meeting.ts

```diff
@@ -325,4 +332,21 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel);
     sendMessageToParent('meeting.getAppContentStageSharingState', callback);
   }
+
+  /**
+   * Registers a handler for changes to paticipant speaking states. If any participant is speaking, isSpeakingDetected
+   * will be true. If no participants are speaking, isSpeakingDetected will be false. Only one handler can be registered
+   * at a time. A subsequent registration replaces an existing registration.
+   * @param handler The handler to invoke when the speaking state of any participant changes (start/stop speaking).
+   */
+  export function registerDetectSpeakingStateChangedHandler(handler: (speakingState: ISpeakingState) => void): void {
+    if (!handler) {
+      throw new Error('[registerDetectSpeakingStateChangedHandler] Handler cannot be null');
+    }
+    ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
+    registerHandler('meeting.speakingStateChanged', (speakingState: ISpeakingState) => {
+      ensureInitialized();
```

Comment:
I don't think this second ensureInitialized call is necessary. Initialization of the SDK is only ever done one time and for this handler to be invoked, the SDK must already have been initialized since you checked that on line 346.